docs/manually-deploying-atmosphere-ansible.md

```diff
@@ -21,122 +21,109 @@ The latest code (usually `master`) for Atmosphere Ansible.
 * Repository:
 
 	```
-	git clone https://github.com/jetstream-cloud/atmosphere-ansible.git /opt/dev/testing
+	git clone https://github.com/jetstream-cloud/atmosphere-ansible.git repo-clone
 	```
 
 ## Set up Environment
 
 1. Change to code directory
 
 	```
-	cd <path-to-repo-clone>/ansible
+	cd <path-to-repo-clone>
 	```
-	
-1. Verify correct `roles_path` and `fact_cache` time is used for test environment, defined in `ansible.cfg`
+
+1. Copy `variables.ini.dist` to `variables.ini` and populate configuration variables. This example uses relative path names from the repository root:
 
 	```
-	...
-	roles_path    = roles			# Uses local path.  Sometimes is set to full path to roles
-	...
-	fact_caching = redis			# Ensure that redis is installed and running
-	fact_caching_timeout = 14400	# Set to at least 4 hours.  Default is 24 hours
-	...
+	[COMMON]
+	ATMOSPHERE_ANSIBLE_DIR = .
+	ATMOSPHERE_ANSIBLE_LOG_DIR = logs
+
+	[ansible.cfg]
+	ANSIBLE_FACT_CACHE_BACKEND = redis
+	ANSIBLE_FACT_CACHE_TIMEOUT = 14400
+	ANSIBLE_MANAGED_STR = Ansible managed: {file} modified on %Y-%m-%d %H:%M:%S by {uid} on {host}
+	ANSIBLE_SSH_TIMEOUT = 10
+
+	SUBSPACE_PLUGINS_DIR = ; /opt/env/atmo/lib/python2.7/site-packages/subspace/plugins
+	SUBSPACE_CALLBACK_WHITELIST = ; play_logger
+	SUBSPACE_NO_COWS = ; 1 or 0 , 0 = Cows and 1 = No Cows
+	SUBSPACE_COW_SELECTION = ; default (NOTE: Only if NO_COWS == 0)
 	```
-	
-1. Use existing `hosts` and `group_vars` or create them
+
+1. Create logs directory in `<path-to-repo-clone>/ansible`
+
+1. Use existing `hosts` and `group_vars`, if available locally, or create them
 
 	```
 	ln -s /opt/dev/atmosphere-ansible/ansible/hosts hosts
 	ln -s /opt/dev/atmosphere-ansible/ansible/group_vars group_vars
 	```
-	
+
 	OR
-	
-	Clone and modify files from: <https://github.com/iPlantCollaborativeOpenSource/clank/tree/master/dist_files> into `<path-to-repo-clone>/ansible/`
-	
+
+	- Clone group_vars/ and hosts from: <https://github.com/iPlantCollaborativeOpenSource/clank/tree/master/dist_files> into `<path-to-repo-clone>/ansible/`
+	- Rename `hosts.dist` to `hosts`
+
 1. Basic testing structure for Ansible hosts files and vars
 
 	```
 	## Set up testing group_vars for one cloud
 	cd group_vars
-	ln -s atmosphere testing 		# Must be done for each cloud environment
+	ln -s atmosphere-provider1 testing 		# Must be done for each cloud environment
 	cd ..
 
 	# Set up testing group_vars for multiple clouds
 	cd group_vars
-	ln -s atmosphere testing1 		# Must be done for each cloud environment
-	ln -s atmosphere testing2 		# Must be done for each cloud environment
+	ln -s atmosphere-provider1 testing1 		# Must be done for each cloud environment
+	ln -s atmosphere-provider2 testing2 		# Must be done for each cloud environment
```

Comment:
Thank you for clarifying this! 

The instruction tripped me up, I thought it was referring this `[atmosphere:children]` previously ([ref](https://github.com/iPlantCollaborativeOpenSource/clank/blob/master/dist_files/hosts.dist#L91-L93)).